### ISSUE TITLE
Trailing `0x00` bytes on the `/base64` endpoint

### ISSUE THREAD

**Issue by mpdude** [mccutchen/go-httpbin#118]
When I request `/base64/dGVzdC1pbWFnZQ==`, I get a 12 byte response back, including two trailing `0x00` bytes. IMHO, `dGVzdC1pbWFnZQ==` should decode to `test-image`, no trailing 0-bytes.

```
curl -i  http://localhost:8080/base64/dGVzdC1pbWFnZQ== | hexdump -C
00000000  48 54 54 50 2f 31 2e 31  20 32 30 30 20 4f 4b 0d  |HTTP/1.1 200 OK.|
00000010  0a 41 63 63 65 73 73 2d  43 6f 6e 74 72 6f 6c 2d  |.Access-Control-|
00000020  41 6c 6c 6f 77 2d 43 72  65 64 65 6e 74 69 61 6c  |Allow-Credential|
00000030  73 3a 20 74 72 75 65 0d  0a 41 63 63 65 73 73 2d  |s: true..Access-|
00000040  43 6f 6e 74 72 6f 6c 2d  41 6c 6c 6f 77 2d 4f 72  |Control-Allow-Or|
00000050  69 67 69 6e 3a 20 2a 0d  0a 43 6f 6e 74 65 6e 74  |igin: *..Content|
00000060  2d 54 79 70 65 3a 20 74  65 78 74 2f 70 6c 61 69  |-Type: text/plai|
00000070  6e 0d 0a 44 61 74 65 3a  20 46 72 69 2c 20 30 35  |n..Date: Fri, 05|
00000080  20 4d 61 79 20 32 30 32  33 20 31 36 3a 35 35 3a  | May 2023 16:55:|
00000090  32 37 20 47 4d 54 0d 0a  43 6f 6e 74 65 6e 74 2d  |27 GMT..Content-|
000000a0  4c 65 6e 67 74 68 3a 20  31 32 0d 0a 0d 0a 74 65  |Length: 12....te|
000000b0  73 74 2d 69 6d 61 67 65  00 00                    |st-image..|
000000ba
```

**Comment by mccutchen** [comment]
Ahhh good catch, thanks for the report!  I've got a fix in https://github.com/mccutchen/go-httpbin/pull/119.  Turns out we were not accounting for padding when calculating the decoded slice length (and also turns out there's a simpler way to do this given an input string).

**Comment by mccutchen** [comment]
Fix released in v2.8.0:
https://github.com/mccutchen/go-httpbin/releases/tag/v2.8.0

**Comment by mpdude** [comment]
Phew, awesome response time! Thank you!
